controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    gripper_controller:
      type: joint_trajectory_controller/JointTrajectoryController

arm_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
    command_interfaces:
      - position
    state_interfaces:
      - position

gripper_controller:
  ros__parameters:
    joints:
      - joint6a
      - joint6b
    command_interfaces:
      - position
    state_interfaces:
      - position